.github/workflows/lint.yml

```diff
@@ -0,0 +1,38 @@
+name: Lint
+
+on:
+    push:
+        branches:
+            - master
+            - release/*
+    pull_request:
+        branches:
+            - master
+            - release/*
+
+jobs:
+    run:
+        runs-on: ubuntu-latest
+        strategy:
+            matrix:
+                php: ['5.6'] #'7.0', '7.1', '7.2', '7.3', '7.4']
+        name: PHP ${{ matrix.php }}
+        steps:
+            -   name: Checkout
+                uses: actions/checkout@v2
+
+            # https://github.com/shivammathur/setup-php
+            -   name: Setup
+                uses: shivammathur/setup-php@v2
+                with:
+                    php-version: ${{ matrix.php }}
+                    coverage: xdebug
```

Comment:
Deze zou op `none` kunnen staan, toch? Waar heb je xdebug voor nodig?

Reply:
Juist niet, itertools heeft 100% code coverage, en daar heb ik xdebug voor nodig.  Dat is ook de reden dat in composer.json staat: `phpunit --configuration phpunit.xml.dist && coverage-check coverage.xml 100`

Reply:
Ohja, dat was de reden dat ik op xdebug was gebleven: `PCOV needs PHP >= 7.1.`.  En itertools is nog compatible met php 5.6.

Ik wil heel graag een nieuwe major versie van itertools maken met een heleboel deprecated code minder en wat php 7.2+ reatures.  Maar voor nu is het dus nog xdebug